src/browser-detector/BrowserDetector.d.ts

```diff
@@ -0,0 +1,12 @@
+import { PureComponent } from 'react';
+
+type Browser = 'chrome' | 'firefox' | 'safari' | 'msie' | 'msedge' | 'opera';
+
+export type BrowserDetectorProps = {
+  children: any, // TODO: t.ReactChildren
+  placeholder: (detectedBrowser: string) => any, // TODO: t.ReactChildren
```

Comment:
https://github.com/lancedikson/bowser/blob/master/typings.d.ts doesn't seem to be very interesting :(